### ISSUE TITLE
Repeated graphic with empty url

### ISSUE THREAD

**Issue by hackmanschorsch** [dariok/page2tei#29]
If a TranskribusMetadata value is in the PAGE XML a second graphic is produced, in this case with an empty url, because no imgUrl was given. Maybe this can be skipped then?

 <surface lrx="3245" lry="4949" ulx="0" uly="0" xml:id="facs_8">
         <graphic height="4949px" url="DSC04916.JPG" width="3245px"/>
         <graphic height="4949px" url="" width="3245px"/>

**Comment by dariok** [comment]
Certainly. This will be handled in the next release with the linked PR.
